backend/todolist/src/main/java/team10/todolist/domain/Board.java

```diff
@@ -0,0 +1,82 @@
+package team10.todolist.domain;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Column;
+import team10.todolist.dto.BoardDto;
+
+import java.time.LocalDateTime;
+
+public class Board {
+
+    @Id
+    private Long id;
+    private String title;
+    private String contents;
+    private String category;
+
+    @Column(value = "date_time")
+    private LocalDateTime dateTime;
+    @Column(value = "is_deleted")
+    private int isDeleted = 0;
```

Comment:
```suggestion
    @Column(value = "date_time")
    private LocalDateTime dateTime;
    
    @Column(value = "is_deleted")
    private int isDeleted = 0;
```
- 무슨 dateTime인가요? 확실하게 적어주세요.
- deleted인데 왜 int 타입인가요?